MeinFiskal_productdescription_added

wrueting opened the pull request:
MeinFiskal is added to the productdescription for DE market.

wrueting committed:
Wording

wrueting committed:
MeinFiskal added to the product overview, description and step-by-step guide

wrueting committed:
media paths corrected

wrueting committed:
Screenshot Password Change Success changed

wrueting committed:
Added Link to Portal Download Area

technical-writer-fabry committed:
Update meinFiskal.md

spelling and wording

technical-writer-fabry reviewed (commented):
done some spelling and wording, the other part is perfect

ft-kha committed:
AKO

Ich habe da schon einen Punkt, den ich hier gerne ansprechen möchte: Wieso bringen wir das AKO hier mit rein?
1.  gibt es das AKO noch gar nicht 
2. thema datenquelle (für mich der wichtiger punkt): posarchiv und meinfiskal haben als datenquelle die middleware.
AKO hat als datenquelle den manuellen upload (ako.fiskaltrust.de) und ist so mit den anderen produkten nicht vergleichbar. AKO ist mehr eine dropbox für fiskaldaten.

Außerdem stimmen die 12 jahre nicht, das sind immer 10 jahre (auch wenn manchmal 12 jahre stehen - die geschichte dahinter erzähle ich Dir gerne mal).

wrueting committed:
change AKO description from 12 years to 10 years

wrueting committed:
changed ako description 12 years to 10 years
